### ISSUE TITLE
Should probably reference the nuget package, not the local dll, on the website?

### ISSUE THREAD

**Issue by WhiteBlackGoose** [fsprojects/FSharpPlus#438]
I've seen a sample on the website, which looks like this:
```fs
#r @"../../src/FSharpPlus/bin/Release/net45/FSharpPlus.dll"
open FSharpPlus
```
But it's probably a good idea to show real life usage, like this:
```fs
#r "nuget:FSharpPlus"
open FSharpPlus
```
And for developers, it would be 
```fs
#i "MyPath/Debug/bin"
#r "nuget:FSharpPlus"
open FSharpPlus
```

But I didn't get how to modify the website, so opened an issue.

**Comment by wallymathieu** [comment]
I agree. The docs should be rendered so that the ref paths show.

**Comment by smoothdeveloper** [comment]
#435 is pending merge which address the first part.

**Comment by wallymathieu** [comment]
You could add a lot of : 
```
    (**
    ```f#
    #r @"nuget: FSharpPlus"
    ```
    *)
```
et.c.

**Comment by gusty** [comment]
Implemented in #489
